.github/workflows/test_unit.yaml

```diff
@@ -179,26 +181,21 @@ jobs:
       - name: Add msbuild to PATH
         uses: microsoft/setup-msbuild@v1.1
 
-      - name: Install glean depedencies
+      - name: Install depedencies
         shell: bash
         run: |
           pip3 install -r requirements.txt
-          python3 scripts/utils/generate_glean.py
+          pip3 install oathtool
+          python3 -m oathtool.generate-script
 
-      - name: Importing translation files
+      - name: Building tests
         shell: bash
         run: |
           export PATH=/c/MozillaVPNBuild/bin:$PATH
-          python3 scripts/utils/import_languages.py -m
-
-      - name: Installing the oathtool
-        shell: bash
-        run: |
-          pip3 install oathtool
-          python3 -m oathtool.generate-script
+          mkdir -p build
+          cmake -S . -B $(pwd)/build
+          cmake --build $(pwd)/build --config Release --target unit_tests
 
       - name: Running tests
         shell: bash
-        run: |
-          export PATH=/c/MozillaVPNBuild/bin:$PATH
-          MVPN_OATHTOOL=oathtool ./scripts/tests/unit_tests.sh
```

Comment:
This is more than unit_tests. It runs:
- unit-tests
- native messaging tests
- auth unit tests

Reply:
I have added a `CMakeLists.txt` for the missing native messaging tests, and I have added some `CTest` commands to run the tests. Hopefully that should ensure we are getting test coverage.